shared/app/src/commonMain/kotlin/com/adammcneilly/pocketleague/shared/app/feed/FeedPresenter.kt

```diff
@@ -31,12 +31,16 @@ class FeedPresenter(
             mutableStateOf(emptyList<MatchDetailDisplayModel>())
         }
 
-        var events by remember {
+        var ongoingEvents by remember {
+            mutableStateOf(emptyList<EventGroupDisplayModel>())
+        }
+
+        var upcomingEvents by remember {
             mutableStateOf(emptyList<EventGroupDisplayModel>())
         }
 
         LaunchedEffect(Unit) {
-            async {
+            launch {
```

Comment:
I think this gives us race conditions? Maybe we should just wait for all calls to complete.